test-tests/shared/src/test/scala/zio/test/AssertionSpec.scala

```diff
@@ -67,7 +67,7 @@ object AssertionSpec
             sampleUser,
             isCase[SampleUser, (String, Int)](
               termName = "SampleUser",
-              SampleUser.unapply,
+              { case SampleUser(name, age) => Some((name, age)) },
```

Comment:
This is Dotty related too?

Reply:
Yes. Explicit calls to `unapply` no longer return an `Option`. See https://github.com/lampepfl/dotty/issues/2335.